lib/paperclip/io_adapters/uri_adapter.rb

```diff
@@ -53,7 +53,7 @@ def default_filename
     def download_content
       options = { read_timeout: Paperclip.options[:read_timeout] }.compact
 
-      open(@target, **options)
+      self.open(@target, options)
```

Comment:
This change does not address the Kernel.open warning so I have not cherry-picked it in #2